docs/integrations/databases/opentelemetry/elasticsearch-opentelemetry.md

```diff
@@ -0,0 +1,252 @@
+:::
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.  
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ElasticSearch-OpenTelemetry/ElasticSearch-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml required for Elasticsearch Collection.
+
+Below are the input required:
+
+-   **Endpoint** - Enter the url of the server which needs to be monitored. Example: http://localhost:9200
+-   **User Name** - Enter the Elasticsearch Username.
+-   **Elasticsearch cluster log path** : By default, Elasticsearch logs are stored in `/var/log/elasticsearch/ELK-<Clustername>.log`.
+-   **Tags** : `db.cluster.name`.
+
+You can include additional metadata which you want to be tagged along with the data ingested to Sumo Logic.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+For linux platform - Click on **Download Environment Variables File** button to get the file with the password which is supposed to be set as environment variable.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/ElasticSearch-OpenTelemetry/ElasticSearch-YAML.png')} alt="YAML" />
+
+### Step 3: Sending logs and metric to Sumo
+
+Once you have the yaml file downloaded in step 2, follow the below steps based on the platform of the machine :
+
+<Tabs
+  className="unique-tabs"
+  defaultValue="Linux"
+  values={[
+    {label: 'Linux', value: 'Linux'},
+    {label: 'Windows', value: 'Windows'},
+    {label: 'macOS', value: 'macOS'},
+  ]}>
+
+<TabItem value="Linux">
+
+1.  Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the Elasticsearch instance which needs to be monitored.
+2.  Place Env file in the following directory
+```sh
+/etc/otelcol-sumo/env/
+```
+3.  restart the collector using
+```sh
+ sudo systemctl restart otelcol-sumo
+```
+
+</TabItem>
+<TabItem value="Windows">
+
+1.  Copy the yaml file to `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\conf.d` folder in the machine which needs to be monitored.
+2.  Restart the collector using 
+```sh
+Restart-Service -Name OtelcolSumo
+```
+
+</TabItem>
+<TabItem value="macOS">
+
+1.  Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the Elasticsearch instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command 
+```sh
+ otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+```
+
+</TabItem>
+</Tabs>
+
+After successful execution of the above command, Sumo will start receiving the data from your host machine. This will install the app to your Sumo Logic Org. 
```

Comment:
```suggestion
After successful execution of the above command, Sumo will start receiving the data from your host machine. 
Click **Next**. This will install the app to your Sumo Logic Org. 
```